src/libraries/System.Net.Http.Json/tests/FunctionalTests/HttpClientJsonExtensionsTests.cs

```diff
@@ -580,6 +580,25 @@ await foreach (string s in client.GetFromJsonAsAsyncEnumerable<string>("http://d
                 await Task.Delay(TimeSpan.FromMilliseconds(10));
             }
         }
+
+        [Fact]
+        public async Task GetFromJsonAsAsyncEnumerable_SerializerCaseInsensitiveTest()
```

Comment:
Technically web defaults use camel casing, not case insensitive binding.
```suggestion
        public async Task GetFromJsonAsAsyncEnumerable_SerializerUsesCamelCase()
```